tests/rtf/NetworkClock/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright: (C) 2017 iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Silvio Traversaro <silvio.traversaro@iit.it>
+# Copy Policy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+add_definitions(-DSHLIBPP_FILTER_API)
+rtf_add_plugin(NetworkClockTest SOURCES NetworkClockTest.cpp)
+# add_library(NetworkClockTest MODULE NetworkClockTest.cpp)
+target_link_libraries(NetworkClockTest YARP_init YARP_OS YARP_rtf ${RTF_LIBRARIES})
+rtf_add_suite(NAME NetworkClockTest
+              FIXTURE rtf_fixturemanager_yarpserver --silent
+              FIXTURE rtf_fixturemanager_yarpmanager --fixture ${CMAKE_CURRENT_SOURCE_DIR}/fixture-network-clock.xml
+              TEST NetworkClockTest TYPE dll
```

Comment:
This should be `RTF_FIXTURE` and `RTF_TEST`

Reply:
Done in https://github.com/robotology/yarp/pull/1438/commits/e66dbc6164f14a8097ca868c6bfca5c8827a9458 .